.github/workflows/foreman_plugin.yml

```diff
@@ -29,9 +29,13 @@ jobs:
     steps:
       - name: Build test matrix
         id: build_matrix
-        # TODO: Make this dynamic, based on the Foreman version
         run: |
-          echo 'matrix={"ruby": ["2.7"], "node": ["12"]}' >> "$GITHUB_OUTPUT"
+          {
+            echo 'matrix<<EOF'
+            #curl --silent --show-error --fail https://raw.githubusercontent.com/theforeman/foreman/${{ inputs.foreman_version }}/.github/matrix.json
+            curl --silent --show-error --fail https://raw.githubusercontent.com/evgeni/foreman/enter-the-matrix/.github/matrix.json
```

Comment:
this needs the correct URL, once the PR is merged ;-)

Reply:
Merged that PR. We should cherry pick it though.

Reply:
does 3.6+ sound reasonable?

Reply:
Right now we don't use it for anything yet, so I'd say so. If any plugin starts using it and needs an older version, we can change it.